Doc/library/math.rst

```diff
@@ -248,7 +248,7 @@ Floating point arithmetic
 
 .. function:: fmod(x, y)
 
-   Return ``fmod(x, y)``, as defined by the platform C library. Note that the
+   Return the floating-point remainder of ``x / y``, as defined by the platform C library function ``fmod(x, y)``. Note that the
```

Comment:
```suggestion
   Return the floating-point remainder of ``x / y``,
   as defined by the platform C library function ``fmod(x, y)``. Note that the
```

Is the diff much larger if you do something like this? because now the line is effectively > 80-chars (it's roughly 130 chars I think)

Reply:
@picnixz I would approve that but Ill see what @skirpichev has to say so that I don't waste my time. In my opinion some formatting is defiantly necessary.